packages/translation/src/lang/en.json

```diff
@@ -1760,6 +1760,20 @@
   },
   "board": {
     "action": {
+      "duplicate": {
+        "title": "Duplicate board",
+        "message": "This will duplicate the board {name} with all its content. If widgets on it contains certain integrations, you are not allowed to use, they will be removed.",
```

Comment:
```suggestion
        "message": "This will duplicate the board {name} with all its content. If widgets reference integrations, that you are not allowed to use, they will be removed.",
```